HTC/U12Plus/AndroidManifest.xml

```diff
@@ -6,5 +6,5 @@
                 android:requiredSystemPropertyName="ro.vendor.build.fingerprint"
 		android:requiredSystemPropertyValue="+htc/ime*"
 		android:priority="97"
-		android:isStatic="true" />
+		android:isStatic="false" />
```

Comment:
Why? We need it to be static.

Reply:
Dil3ma told me to make it dynamic I don't know why I just try anything I being told to try to make it work

Reply:
It just needs to be active. We'll try to handle the rest then, but first you need to clean up this overlay.

Reply:
He sent me one that was active after the command he told me he said he will PR to phh